genologics/entities.py

```diff
@@ -510,7 +510,7 @@ def create_in_memory_instance(cls, lims, container, position, **kwargs):
         """
         Creates a request for creating a single sample object
         """
-        udfs = kwargs.pop("udfs", list())
+        udfs = kwargs.pop("udfs", dict())
```

Comment:
This is interesting! Is it a fix from you or 2to3?

Reply:
This is a manually change by me. It makes a test pass.